x-pack/plugins/ingest_manager/README.md

```diff
@@ -25,7 +25,7 @@ Also you need to configure the hosts your agent is going to use to comunication
 
 ### Getting started
 
-See the Kibana docs for [how to set up your dev environment](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#setting-up-your-development-environment), [run Elasticsearch](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#running-elasticsearch), and [start Kibana](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#running-kibana)
+See the Kibana docs for [how to set up your dev environment](https://www.elastic.co/guide/en/kibana/master/development-getting-started.html)
```

Comment:
This link points to master.  You'll need to manually update the backport PR (e.g. to link to https://www.elastic.co/guide/en/kibana/7.9/development-getting-started.html instead).

If you do make it version-specific, you'll need to update it again after 7.10 splits from 7.x.